project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectService.java

```diff
@@ -288,954 +974,677 @@ Flux<ByteBuffer> roCrateSummary(String projectId)
-   * {@link #update(ExperimentUpdateRequest)}
+   * Tries to find the exact matching {@link OntologyTerm} for a given {@link Curie}.
+   * <p>
+   * If no matching {@link OntologyTerm} can be found, the {@link Mono#empty()} will complete
+   * without emitting a value.
+   * <p>
+   * <b>Exceptions</b>
+   * <p>
+   * Exceptions are wrapped as {@link Mono#error(Throwable)} and are one of the types described in
+   * the throw section below.
    *
-   * @param confoundingVariables the variable information
+   * @param value the {@link Curie} of the term to search for
+   * @return a {@link Mono} emitting the {@link OntologyTerm} if an exact match was found or else
+   * completes empty.
+   * @throws RequestFailedException if the request was not successfully executed
+   * @since 1.10.0
    */
-  record ConfoundingVariableAdditions(
-      List<ConfoundingVariableInformation> confoundingVariables) implements
-      ExperimentUpdateRequestBody {
-
-    public ConfoundingVariableAdditions {
-      confoundingVariables = List.copyOf(confoundingVariables);
-    }
-
-    @Override
-    public List<ConfoundingVariableInformation> confoundingVariables() {
-      return List.copyOf(confoundingVariables);
-    }
-  }
+  Mono<OntologyTerm> getTermWithCurie(Curie value);
 
   /**
-   * A list of confounding variable information for variable update. Can be used in
-   * {@link #update(ExperimentUpdateRequest)}
+   * Requests a {@link Flux} of {@link OntologyTerm} for a given search value.
+   * <p>
+   * The implementation must support:
+   * <ul>
+   *   <li>pagination based on the provided values for offset and limit</li>
+   *   <li>taxa as emitted values, and <strong>taxa only</strong></li>
+   * </ul>
+   * <p>
+   * <b>Exceptions</b>
+   * <p>
+   * Exceptions are wrapped as {@link Flux#error(Throwable)} and are one of the types described in
+   * the throw section below.
    *
-   * @param confoundingVariables the variable information
+   * @param value   the value for searching matching {@link OntologyTerm}
+   * @param offset  the offset value from 0 for paginated queries
+   * @param limit   the maximum number of hits returned in the flux
+   * @param sorting a {@link List} of {@link SortOrder}s to sort the results by
+   * @return a {@link Flux} of {@link OntologyTerm} matching the search value
+   * @throws RequestFailedException if the request was not successfully executed
+   * @since 1.10.0
    */
-  record ConfoundingVariableDeletions(
-      List<ConfoundingVariableInformation> confoundingVariables) implements
-      ExperimentUpdateRequestBody {
-
-    public ConfoundingVariableDeletions {
-      confoundingVariables = List.copyOf(confoundingVariables);
-    }
-
-    @Override
-    public List<ConfoundingVariableInformation> confoundingVariables() {
-      return List.copyOf(confoundingVariables);
-    }
-  }
+  Flux<OntologyTerm> getTaxa(String value, int offset, int limit, List<SortOrder> sorting);
```

Comment:
What is the difference of `getTaxa` and `getTerms`?

Reply:
It is described in the Java documentation of the methods :)

Reply:
I don't see it? Ah the `taxa only` part